package.json

```diff
@@ -21,8 +21,10 @@
   "homepage": "https://github.com/Manvityagi/Opportunity-Calendar-Backend#readme",
   "dependencies": {
     "body-parser": "^1.19.0",
+    "cluster": "^0.7.7",
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
+    "esm": "^3.2.25",
```

Comment:
why do we need this package?

Reply:
Sorry removed it.
It was added by mistake.